python/cudf/cudf/core/column/numerical.py

```diff
@@ -405,10 +406,10 @@ def can_cast_safely(self, to_dtype):
                 return False
 
 
+@annotate("CUDF_BINARY_OP", color="orange", domain="cudf_python")
```

Comment:
The `CUDF_` prefix isn't necessary in the message since these things are already in the cuDF domain.